values.yaml

```
@@ -81,3 +81,7 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+storage_class: 
+  access_mode: ReadWriteOnce
```

technically access mode is not part of storage class, but of the volume, so maybe it's better to make differently for clarity.
```suggestion
persistence:
  storage_class_name: vspherecsi-new
  access_mode: ReadWriteOnce
```